packages/graph/src/Graph.ts

```diff
@@ -944,6 +977,11 @@ export class Graph extends SVGZoomWidget {
 Graph.prototype._class += " graph_Graph";
 Graph.prototype.implements(IGraph.prototype);
 Graph.prototype.implements(ITooltip.prototype);
+export interface Graph {
+    useDetailedTooltip(): boolean;
+    useDetailedTooltip(_: boolean): this;
+}
+Graph.prototype.publish("useDetailedTooltip", false, "boolean");
```

Comment:
This paradigm doesn't work - as the whole purpose of the tooltipHtml function is to allow the user to provide their own.

Reply:
Can you add a description and move it so its below "snapToGrid".

Also will you rebase onto latest candidate-2.x.x as I think it won't merge correctly as it stands (I am bit surprised that GitHub isn't warning about it...